cve_bin_tool/extractor.py

```diff
@@ -86,7 +108,20 @@ def extract_file_rpm(cls, filename, extraction_path):
     def extract_file_deb(cls, filename, extraction_path):
         """ Extract debian packages """
         if not inpath("ar"):
-            print("Error: need 'ar' binary in path")
+            """ Acutally MinGW provides ar, so this might never get called """
+            archive = unix_ar.open(filename, "r")
+            archive.extractall(extraction_path)
+            archive.close()
+            datafile = glob.glob(os.path.join(extraction_path, "data.tar.*"))[0]
+            if not inpath("tar"):
+                tar = tarfile.open(datafile)
```

Comment:
I think that [shutil.unpack_archive](https://docs.python.org/3/library/shutil.html#shutil.unpack_archive) might be a one-liner to replace the tar and zipfile usages